cmsig/seal-symfony-bundle/0.8/config/packages/cmsig_seal.yaml

```
@@ -0,0 +1,9 @@
+cmsig_seal:
+    index_name_prefix: '%env(default::TEST_INDEX_PREFIX)%'
+    schemas:
+        default:
+            dir: '%kernel.project_dir%/config/schemas'
+            engine: default
+    engines:
+        default:
+            adapter: '%env(SEAL_DSN)%'

```

would gow ith the `index_name_prefix` the same as doctrine.yaml does only define it in the test env and also keep the paratest parameter in mind to avoid sideeffects:

```suggestion
cmsig_seal:
    schemas:
        default:
            dir: '%kernel.project_dir%/config/schemas'
            engine: default
    engines:
        default:
            adapter: '%env(SEAL_DSN)%'

when@test:
    cmsig_seal:
        # "TEST_TOKEN" is typically set by ParaTest
        index_name_prefix: 'test_%env(default::TEST_TOKEN)%'
```